File: Locadora.Domain/Streaming.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Locadora.Domain
{
    public class Streaming
    {
        public int StreamingId { get; set; }
        public string Sinopse { get; set; }
        public string Titulo { get; set; }
        public int ClassificacaoIndicativa { get; set; }
        public string ImagemURL { get; set; }
        public DateTime DataLancamento { get; set; }
        public DateTime DataCadastro { get; set; }
        public DateTime DataModificado { get; set; }
        public int TipoStreamingId { get; set; }
        public TipoStreaming TipoStreaming { get; set; }
        public List<StreamingAtor> StreamingAtor { get; set; }
        public List<GeneroStreaming> GeneroStreaming { get; set; }

    }
}

File: Locadora-APP/src/app/_models/Streaming.ts
import { TipoStreaming } from './TipoStreaming';
import { Ator } from './Ator';
import { Genero } from './Genero';

export interface Streaming {
    id: number;
    sinopse: string;
    titulo: string;
    classificacaoIndicativa: number;
    imagemURL: string;
    dataLancamento: Date;
    dataCadastro: Date;
    dataModificado: Date;
    tipoStreaming: TipoStreaming;
    genero: Genero[];
    ator: Ator[];
}
